web/rootfs/defaults/settings-config.js

```diff
@@ -224,6 +229,15 @@ config.peopleSearchQueryTypes = ['user','conferenceRooms'];
 // Prejoin page.
 config.prejoinPageEnabled = {{ $ENABLE_PREJOIN_PAGE }};
 
+// Welcome page.
+config.enableWelcomePage = {{ $ENABLE_WELCOME_PAGE }};
+
+// Close page.
+config.enableClosePage = {{ $ENABLE_CLOSE_PAGE }};
+
+// Noisy mic detection notification.
+config.enableNoisyMicDetection = {{ $ENABLE_NOISY_MIC_DETECTION }};
```

Comment:
This probably belongs in the audio section. We don't have a rule per se, but I tried to group the options semantically, as a best effort.

Reply:
Is there anything else? I can do it all at once